### ISSUE TITLE
Circular dependency detected: TOP => db:create => db:load_config => environment => db:load_config

### ISSUE THREAD

**Issue by deepj** [rosenfeld/active_record_migrations#23]
I'm trying to upgrade one of our Grape API apps with ActiveRecord 5.2 where we also use ActiveRecord Migrations. Now, we're getting the following error

```
# rake db:create
rake aborted!
Circular dependency detected: TOP => db:create => db:load_config => environment => db:load_config
/usr/local/bundle/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:create => db:load_config => environment
(See full trace by running task with --trace)
```

**Comment by rosenfeld** [comment]
I believe it should be now fixed in the latest release (5.2.0.1). Give it a try and if it doesn't work please let me know.

**Comment by deepj** [comment]
@rosenfeld Hi, it's fixed. Thanks! 👍
